app/controllers/todos_controller.rb

```diff
@@ -1 +1,39 @@
-# This controller is for all the CRUD operations related to a Todo.
\ No newline at end of file
+
+#Home page for the To Do list program. Will display the To Do list.
+#Each todo will be An Array object. The .erb page will have to use an each do loop to iterate through each object and display the title-description-completed. Not sure how to use the id column quite yet. Do I need to capture a session id to make a new to do on this list?  
+
+MyApp.get "/todos/todo_home" do
+
+  @todos = Todo.all
+
+  erb :"/todos/todo_home"
+end
+
+
+#This controller action will make the new todo form available to the user. 
+MyApp.get "/todos/todos_new" do
+
+
+  erb :"todos/todos_new"
+end
+#This controller action will process the new todo form to create a record of a new todo.
+MyApp.post "/todos/todos_create" do
+  @todo = Todo.new
+  @todo.title = (params[:title])
+  @todo.description = (params[:description])
+  @todo.completed = (params[:completed])
+  @todo.id #is this column for the session id?
```

Comment:
> `@todo.id #is this column for the session id?`

No. You never need to set the value of some row's `id` column manually. ActiveRecord will handle that for us.